### PR TITLE
GCP deprecations

### PR DESCRIPTION
**Pull request by rambleraptor**
##### SUMMARY

Deprecating more GCP modules.

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/porting_guides/porting_guide_2.8.rst [hunk 1]
```diff
@@ -228,7 +228,13 @@ The following modules will be removed in Ansible 2.12. Please update your playbo
 * ``katello`` use <https://github.com/theforeman/foreman-ansible-modules> instead.
 * ``github_hooks`` use :ref:`github_webhook <github_webhook_module>` and :ref:`github_webhook_facts <github_webhook_facts_module>` instead.
 * ``digital_ocean`` use :ref `digital_ocean_droplet <digital_ocean_droplet_module>` instead.
-* ``gce`` use :ref `gce_compute_instance <gce_compute_instance_module>` instead.
+* ``gce`` use :ref `gcp_compute_instance <gcp_compute_instance_module>` instead.
```
Comment (rambleraptor) [inline docs/docsite/rst/porting_guides/porting_guide_2.8.rst:240]: This was referencing a module that does not exist.

**Comment by rambleraptor** [comment]
ptal!

**Comment by gundalow** [comment]
@rambleraptor Hi, can you please rebase so I can mergethis

**Comment by rambleraptor** [comment]
done!

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@rambleraptor Thanks, please create a backport and we will get this merged into `stable-2.8`

Doesn't look like this PR ever got reviewed before, so no `label/shipit` or automerging happened.

**Comment by rambleraptor** [comment]
Thanks! Created #55534 to backport this.

**Comment by samccann** [comment]
@rambleraptor - I generated the 2.8 porting guide in preparation for the upcoming release and the following line points to a module that doesn't exist:
https://github.com/ansible/ansible/blame/devel/docs/docsite/rst/porting_guides/porting_guide_2.8.rst#L247

(in case I'm getting the link wrong - the gcp_target_proxy module says it is replaced by the gcp_compute_target_proxy module, but that module doesn't exist.)

Can you open a pr to fix this? I'm not sure which of the gcp_compute_* modules is the correct replacement.

**Comment by acozine** [comment]
@rambleraptor since the old module [only supported http](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/google/_gcp_target_proxy.py#L34) we've redirected users to `gcp_compute_target_http_proxy`.

**Pull request merged**